code/game/gamemodes/game_mode.dm

```diff
@@ -53,6 +54,8 @@
 	if(!Debug2)
 		if(playerC < required_players)
 			return 0
+		if(max_players && (playerC > max_players))
+			return 2
```

Comment:
Signifies that it should not start unless forced by an admin, hence the above == 1 for the standard secret gamemode selection